documentation/modules/configuring/con-config-kafka-connect.adoc

```diff
@@ -37,27 +37,34 @@ For more information, see xref:con-high-volume-config-properties-{context}[Handl
 .Example `KafkaConnect` custom resource configuration
 [source,yaml,subs=attributes+,options="nowrap"]
 ----
+# Basic configuration (essential)
 apiVersion: {KafkaConnectApiVersion}
 kind: KafkaConnect # <1>
 metadata:
   name: my-connect-cluster
   annotations:
     strimzi.io/use-connector-resources: "true" # <2>
+# Deployment specifications
 spec:
+  # Replicas (essential)
   replicas: 3 # <3>
+  # Authentication (recommended)
   authentication: # <4>
     type: tls
     certificateAndKey:
       certificate: source.crt
       key: source.key
       secretName: my-user-source
+  # Bootstrap servers (essential)
   bootstrapServers: my-cluster-kafka-bootstrap:9092 # <5>
+  # TLS configuration (optional)
   tls: # <6>
     trustedCertificates:
       - secretName: my-cluster-cluster-cert
         pattern: "*.crt"
       - secretName: my-cluster-cluster-cert
         pattern: "*.crt"
+  # Kafka Connect configuration (optional)
```

Comment:
Should we mark this as recommended because of the topic names and group ID that often cause problems with the default values?